servers/game/src/main/kotlin/fr/rob/game/Main.kt

```diff
@@ -70,6 +70,6 @@ class Main : BaseApplication() {
     override fun registerModules(modules: MutableList<AbstractModule>) {
         modules.add(ConfigModule(this))
         modules.add(DatabaseModule(eventManager))
-        modules.add(SecurityModule(setup, processManager))
+        modules.add(SecurityModule(this, setup, processManager))
```

Comment:
In this case, we just need the environment so, replace this by `modules.add(SecurityModule(this.env, setup, processManager))`